midi2audio.py

```diff
@@ -36,17 +36,18 @@
 
 DEFAULT_SOUND_FONT = '~/.fluidsynth/default_sound_font.sf2'
 DEFAULT_SAMPLE_RATE = 44100
+DEFAULT_GAIN = 0.2
 
 class FluidSynth():
     def __init__(self, sound_font=DEFAULT_SOUND_FONT, sample_rate=DEFAULT_SAMPLE_RATE):
         self.sample_rate = sample_rate
         self.sound_font = os.path.expanduser(sound_font)
 
-    def midi_to_audio(self, midi_file, audio_file):
-        subprocess.call(['fluidsynth', '-ni', self.sound_font, midi_file, '-F', audio_file, '-r', str(self.sample_rate)])
+    def midi_to_audio(self, midi_file, audio_file, gain=DEFAULT_GAIN):
```

Comment:
Why it's in `midi_to_audio()`? Is it expected to be different for each file? It can be in `__init__()`.

Reply:
Moved it into __init__() now